src/components/templates/Camp/components/RoomModal.tsx

```diff
@@ -40,7 +40,7 @@ export const RoomModal: React.FC<RoomModalProps> = ({
   room,
   joinButtonText,
 }) => {
-  useFirestoreConnect("venues");
+  useSparkleFirestoreConnect("venues");
```

Comment:
[OOS] Not relevant to this PR, but calling this out as a pattern we want to remove, as we shouldn't ever be loading all of the venues like this.